Lab4.py

```diff
@@ -140,7 +136,7 @@ def temp_kanger(t, t_kanger):
     return temperature
 
 #Appling heat diffusion model to the validation problem:
-x, time, heat = heatdiff(1, 0.2, 0.04, 0.0002)
+x, time, heat = heatdiff(1, 0.2, 0.2, 0.02)
```

Comment:
I understand these are the default values, but I prefer having dx and dt as smaller values because the resolution is better for the solver validation plot.